pi-hole/docker-compose.yml

```diff
@@ -24,3 +25,5 @@ services:
     networks:
       default:
         ipv4_address: $APP_PI_HOLE_IP
+    cap_add:
+      - NET_ADMIN
```

Comment:
Do you know why this capability needs to be added?

Reply:
Yes, It's documented on https://github.com/pi-hole/docker-pi-hole as I explained in https://github.com/getumbrel/umbrel-apps/issues/145.